alpine/5/Dockerfile

```diff
@@ -38,15 +42,31 @@ RUN mv /usr/local/etc/php/php.ini-production /usr/local/etc/php/php.ini
 # https://hub.docker.com/r/mlocati/php-extension-installer/tags
 COPY --from=mlocati/php-extension-installer /usr/bin/install-php-extensions /usr/local/bin/
 
-# Install chromium
 RUN set -eux; \
     apk add --no-cache \
     font-isas-misc \
     supervisor \
     mysql-client \
     chromium \
     ttf-freefont \
-    ttf-dejavu
+    font-noto-cjk \ 
+    ttf-dejavu \
+    bash \
+    && apk add --no-cache --virtual .build-deps \
+    wget \
+    && rm -rf /var/cache/apk/*
+
+# Make sure Chinese fonts installed 
+RUN mkfontscale && mkfontdir && fc-cache -fv
+
+# Setup Chinese Language Env
+RUN echo "export LANG=zh_CN.UTF-8" >> /etc/profile \
+&& echo "export LANGUAGE=zh_CN.UTF-8" >> /etc/profile \
+&& echo "export LC_ALL=zh_CN.UTF-8" >> /etc/profile
```

Comment:
Please remove these lines as they impact all users.

If needed you can pass this through as a configuration